src/module/service.cpp

```diff
@@ -198,6 +199,10 @@ ModuleService_::ModuleService_(std::string addr) {
     module = std::make_shared<Module>(addr);
 }
 
+void ModuleService_::signal_handler(int signum) {
```

Comment:
just wondering, why pass in `signum` if it is then not used?

Reply:
yeah they can take in an unnamed `int`; thanks, @clintpurser.